devops/docker/Groovy/Dockerfile

```diff
@@ -0,0 +1,26 @@
+#################### Using official groovy image ####################
+FROM groovy:jdk11
+#################### End of images used ############################# 
+
+##### Changing user to root to get access to /var/lib/dpkg/lock-frontend ########
+USER root
+##################### End of Change User #############################
+
+##################### Installing Maven and git #######################
+RUN apt-get update \
+    && echo "Installing maven and git dependencies" \
+    && apt-get install --yes --no-install-recommends \
+        git \
+        maven
+##################### End of install #################################
+
+##################### Changing User to Groovy ########################
+USER groovy
+##################### End of Change User##############################
+
+##################### Check install of groovy ########################
+RUN set -o errexit -o nounset \
+    && echo "Testing Groovy installation" \
+    && groovy --version
+##################### End of Dockerfile ##############################
```

Comment:
I believe this (RUN) and previous (USER) steps are not needed, please remove them

Reply:
@strkkk Both these `USER` changes are essential. Without a change to the root user, one cannot get access to the protected files of the container. You may confirm through the below logs. The second one is important since after installing new packages, we need to sure that the entire container runs as expected in non-root environments after all the changes. 

```
$ cat Dockerfile 
#################### Using official groovy image ####################
FROM groovy:jdk11
#################### End of images used ############################# 

##################### Installing Maven and git #######################
RUN apt-get update \
    && echo "Installing maven and git dependencies" \
    && apt-get install --yes --no-install-recommends \
        git \
        maven
##################### End of install #################################

##################### Check install of groovy ########################
RUN set -o errexit -o nounset \
    && echo "Testing Groovy installation" \
    && groovy --version
##################### End of Dockerfile ##############################

$ docker build . -t akmo03/groovy-jdk11:latest
Sending build context to Docker daemon   2.56kB
Step 1/3 : FROM groovy:jdk11
 ---> b55bd63dcb2d
Step 2/3 : RUN apt-get update     && echo "Installing maven and git dependencies"     && apt-get install --yes --no-install-recommends         git         maven
 ---> Running in 1b3c4c387941
Reading package lists...
E: List directory /var/lib/apt/lists/partial is missing. - Acquire (13: Permission denied)
The command '/bin/sh -c apt-get update     && echo "Installing maven and git dependencies"     && apt-get install --yes --no-install-recommends         git         maven' returned a non-zero code: 100
```

Reply:
yes, I understand that. I mean that only last user command (switch to groovy) is not really needed, it is only to run test installation command.

I mean to delete run command and user switch which is just above it

Reply:
Done. I think it is usually a standard to test new images, but we skip through that since we using the official image :)

Reply:
you always can make a local test. Build a new image and just run some command